gateway-js/src/supergraphManagers/UplinkSupergraphManager/loadSupergraphSdlFromStorage.ts

```diff
@@ -135,6 +142,7 @@ export async function loadSupergraphSdlFromStorage({
   const startTime = new Date();
   let result: FetcherResponse;
   try {
+    logger?.debug(`🔧 Fetching supergraph schema from ${endpoint}`);
```

Comment:
I'd be very happy to see these logs expected in tests somewhere

Reply:
This is not to say I was ever good about this in the past.

Reply:
I tend to think of logging as an after-effect for most things, so long as there's a good test for the actual functionality we're logging about (and there is for this stuff at least). So I tend to leave it out of tests. Though I don't feel strongly and happy to add them if there's even a slight preference.

Reply:
With or without is fine by me. My preference towards testing them is just to ensure the code I wrote does what I expect and someone doesn't change that without being intentional about it. I do agree that logs (especially debug) are considerably lower consequence though, and wouldn't block on this.

Reply:
I'm gonna leave out of the tests for now